$OpenBSD: patch-plugins_welcomepage_CMakeLists_txt,v 1.1 2019/11/15 14:30:20 cwen Exp $

Drop useless fix used for gcc-5, because it breaks the build with
ports-gcc>=8.3.0.

Index: plugins/welcomepage/CMakeLists.txt
--- plugins/welcomepage/CMakeLists.txt.orig
+++ plugins/welcomepage/CMakeLists.txt
@@ -28,8 +28,3 @@ target_link_libraries(kdevwelcomepage
     Qt5::QuickWidgets
     KF5::Declarative
 )
-# see https://bugs.launchpad.net/ubuntu/+source/gcc-5/+bug/1568899
-if (UNIX AND CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
-    message(WARNING "Applying workaround for https://bugs.launchpad.net/ubuntu/+source/gcc-5/+bug/1568899")
-    target_link_libraries(kdevwelcomepage gcc_s gcc)
-endif()
